src/app/index.ts

```diff
@@ -37,6 +37,14 @@ export {
   SparklineData
 } from './chart/index';
 
+// CopyToClipboard
+export {
+  CopyModule,
+  CopyService,
+  BlockCopyComponent,
+  InlineCopyComponent
+} from './copy/index';
```

Comment:
I'd like to create modules for each component, so we'll need to add exports for them here as well.

```
export {
  BlockCopyComponent,
  BlockCopyModule,
  CopyService,
  CopyServiceModule,
  InlineCopyComponent,
  InlineCopyModule
} from './copy/index';
```